data/api/tmdb/src/main/java/io/filmtime/data/api/tmdb/TmdbMoviesRemoteSourceImpl.kt

```diff
@@ -84,7 +96,11 @@ internal class TmdbMoviesRemoteSourceImpl @Inject constructor(
     when (val result = apiFunction()) {
       is NetworkResponse.Success -> {
         val videoListResponse = result.body?.results ?: emptyList()
-        Result.Success(videoListResponse.map { it.toVideoThumbnail() })
+        Result.Success(
+          videoListResponse.map {
+            it.toVideoThumbnail()
+          },
+        )
```

Comment:
Do we need this formatting change here? 🤔

Reply:
This is not my change